app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  has_many :posts
+  validates :user_name, unique: true, presence: true,
```

Comment:
- [ ] This line gives error, it's `uniqueness: true` not unique. Please check this out: [uniqueness_validation](https://guides.rubyonrails.org/active_record_validations.html#uniqueness)